net-fs/curlftpfs/curlftpfs-0.9.2.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="Filesystem for accessing FTP hosts"
+DESCRIPTION="A file system for accessing ftp hosts based on FUSE and \
+libcurl.
+As the FTP protocol is not very feature rich, this filesystem does not \
+fulfill every constraint of a real filesystem, but it should be usable \
+for simple tasks like copying and editing files.
+
+CurlFtpFS differentiates itself from other FTP filesystems because it \
+features:
+
+* SSLv3 and TLSv1 support
+* connecting through tunneling HTTP proxies
+* automatically reconnection if the server times out
+* transform absolute symlinks to point back into the ftp file system
+
+Usage on Haiku: First start the userland_fs server:
+	/system/servers/userlandfs_server curlftpfs &
```

Comment:
The kernel automatically runs the server as needed, and that works in the normal case (userlandfs package installed in /system). The workarounds detailed in that ticket are for handling the case where the userlandfs server is installed in /home.

What makes you think it is broken in the normal case? What are the errors given?
The best solution would be, of course, to fix the bugs, instead of hacking around them with launch daemon scripts or instructions in package descriptions.